class-components/src/components/About/About.tsx

```diff
@@ -0,0 +1,24 @@
+import './style.css';
+
+export const About = () => {
+  return (
+    <h3 className="about">
+      The application was made by
+      <a
+        className="about__link"
+        href="https://github.com/aissatsana"
+        target="_blank"
+      >
+        aissatsana
+      </a>
+      as part of the
+      <a
+        className="about__link"
+        href="https://rs.school/courses/reactjs"
+        target="_blank"
+      >
```

Comment:
```suggestion
       <a
        className="about__link"
        href="https://github.com/aissatsana"
        target="_blank"
        rel="noopener noreferrer"
      >
        aissatsana
      </a>
      as part of the
      <a
        className="about__link"
        href="https://rs.school/courses/reactjs"
        target="_blank"
        rel="noopener noreferrer"
      >
        RS School React course
      </a>
```

чтоб линтер не ругался